### PR TITLE
Feature/add auto release changelog

### PR DESCRIPTION
**Pull request by MichaelKohler**

### CODE REVIEW

**Comment by MichaelKohler** [comment]
@freaktechnik r?

File: package.json [hunk 1]
```diff
@@ -58,6 +64,37 @@
...
+    "semantic-release": "^15.13.3"
+  },
+  "release": {
+    "ci": false,
+    "repositoryUrl": "git@github.com:Common-Voice/sentence-collector.git",
+    "plugins": [
+      "@semantic-release/commit-analyzer",
+      "@semantic-release/release-notes-generator",
+      [
+        "@semantic-release/npm",
```
Comment (MichaelKohler) [inline package.json]: mh, I guess we could also just not use that plugin at all and that would be the same?

Comment (freaktechnik) [reply package.json]: that plugin bumps the package.json version, which is used to figure out the next version afaik.

Comment (MichaelKohler) [reply package.json]: oh, right

**Pull request merged**